src/components/Table/styles.styl

```diff
@@ -116,4 +116,7 @@
             padding-bottom .5rem
             font-size .75rem
             color var(--coolGrey)
-            max-width fit-content
+            flex-basis auto
+            flex-shrink 0
+            flex-grow 1
+            max-width none
```

Comment:
For my culture, what was the problem here with max-width fit-content ?

Reply:
It was not working at all in Firefox. Also, MDN marks this value as not production-ready.